aya/src/bpf.rs

```diff
@@ -1068,6 +1068,29 @@ impl Ebpf {
     pub fn programs_mut(&mut self) -> impl Iterator<Item = (&str, &mut Program)> {
         self.programs.iter_mut().map(|(s, p)| (s.as_str(), p))
     }
+
+    /// Enable global statistics tracking for all programs and returns a file descriptor handler.
+    ///
+    /// Statistics tracking will be disabled once the file descriptor is closed or released.
+    ///
+    /// Typical usage:
+    /// 1. Obtain fd from `enable_stats_fd` and bind it to a variable.
+    /// 2. Record the statistic of interest.
+    /// 3. Wait for a recorded period of time.
+    /// 4. Record the statistic of interest again, and calculate the difference.
+    /// 5. Close/release fd.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// # use aya::{Ebpf, EbpfError};
+    /// # use aya_obj::BpfStatsType;
+    /// let _fd = Ebpf::enable_stats_fd(BpfStatsType::RunTime)?;
+    /// # Ok::<(), EbpfError>(())
+    /// ```
+    pub fn enable_stats_fd(stats_type: BpfStatsType) -> Result<OwnedFd, SyscallError> {
+        bpf_enable_stats(stats_type.into())
```

Comment:
As we're not exposing `BpfStatsType` you can just use `bpf_stats_type::BPF_STATS_RUN_TIME` directly